sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Diagnostics/EventProcessorEventSource.cs

```diff
@@ -120,173 +120,18 @@ public virtual void ActiveOwnershipRecordsCount(int count)
             }
         }
 
-        /// <summary>
```

Comment:
Relocated these to `PartitionLoadBalancerEventSource` since they are no longer happening in the `EventProcessorClient` itself.